arch/arm/src/lc823450/lc823450_rtc.c

```diff
@@ -660,7 +660,7 @@ int up_rtc_getrawtime(FAR struct timespec *ts)
   struct timespec now;
   struct timespec diff;
 
-  clock_gettime(CLOCK_MONOTONIC, &now);
+  clock_systime_timespec(&now);
```

Comment:
I think we should keep `clock_gettime(CLOCK_MONOTONIC, &now);` here.

Reply:
Could you give the detail reason?

Reply:
I re-examined the usage again. Seems to be a good change